Qin Warring States/translation.json

```
@@ -89,4 +89,32 @@
   "Wood-button": "Wood",
   "Yan": "Yan",
   "Zhao": "Zhao"
```

```suggestion
  "Zhao": "Zhao",
```
Comma is missing here to turn this into valid JSON.